doc/classes/Color.xml

```diff
@@ -403,26 +404,26 @@
 			<return type="String" />
 			<param index="0" name="with_alpha" type="bool" default="true" />
 			<description>
-				Returns the color converted to an HTML hexadecimal color string in RGBA format (ex: [code]ff34f822[/code]).
-				Setting [param with_alpha] to [code]false[/code] excludes alpha from the hexadecimal string (and uses RGB instead of RGBA format).
+				Returns the color converted to an HTML hexadecimal color [String] in RGBA format, without [code]#[/code].
+				Setting [param with_alpha] to [code]false[/code], excludes alpha from the hexadecimal string, using RGB format instead of RGBA format.
 				[codeblocks]
 				[gdscript]
-				var color = Color(1, 1, 1, 0.5)
-				var with_alpha = color.to_html() # Returns "ffffff7f"
-				var without_alpha = color.to_html(false) # Returns "ffffff"
+				var white = Color(1, 1, 1, 0.5)
+				var with_alpha = white.to_html() # Returns "ffffff7f"
+				var without_alpha = white.to_html(false) # Returns "ffffff"
 				[/gdscript]
 				[csharp]
-				var color = new Color(1, 1, 1, 0.5f);
-				String withAlpha = color.ToHtml(); // Returns "ffffff7f"
-				String withoutAlpha = color.ToHtml(false); // Returns "ffffff"
+				var white = new Color(1, 1, 1, 0.5f);
+				String withAlpha = white.ToHtml(); // Returns "ffffff7f"
+				String withoutAlpha = white.ToHtml(false); // Returns "ffffff"
 				[/csharp]
 				[/codeblocks]
 			</description>
 		</method>
 		<method name="to_rgba32" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the color converted to a 32-bit integer in RGBA format (each byte represents a color channel). RGBA is Godot's default format.
+				Returns the color converted to a 32-bit integer in RGBA format (each component is 8 bits). RGBA is Godot's default format.
```

Comment:
This may be contested. I would love to call them "channels", but the majority of this doc refers to them as "components".

Reply:
That's because channel doesn't fit e.g. hue/saturation/luminance I think. It's just for consistency with different formats.